src/amo/pages/Collection/index.js

```diff
@@ -586,6 +587,12 @@ export class CollectionBase extends React.Component<InternalProps> {
               <meta name="description" content={this.getPageDescription()} />
             </Helmet>
           )}
+          {
+            /* Only include canonical/alternate links if this is a mozilla collection */
+            collection && collection.authorUsername === 'mozilla' && (
```

Comment:
Can we use `mozillaUserId`?

Reply:
Yes that's much better. https://github.com/mozilla/addons-frontend/pull/13129/commits/a323dffb3f3cb08d7a8071139470309df93d0c8e